lisp/lint/declare.el

```diff
@@ -34,10 +39,10 @@
          (errors))
     (eask-lint-first-newline)
     (eask-msg "`%s` with check-declare" (ansi-green file))
-    (setq errors (check-declare-file filename))
+    (setq errors (dlet ((inhibit-message 't)) (check-declare-file filename)))
```

Comment:
Do you mean `let` here? 🤔 Maybe we can use `eask--silent` instead.

Reply:
Nice, I've used that instead. I thought that you needed `dlet` to use `inhibit-message` like that...